src/ledger/LedgerHashUtils.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
```

Comment:
Actually you already have it in this order, but GitHub isn't displaying it in the right order because of the commit date. Please update the commit dates.